pom.xml

```diff
@@ -338,6 +339,10 @@
               <pattern>com.fasterxml.jackson</pattern>
               <shadedPattern>com.spotify.docker.client.shaded.com.fasterxml.jackson</shadedPattern>
             </relocation>
+            <relocation>
+              <pattern>com.google.guava</pattern>
```

Comment:
I think this should be `com.google.common`

see https://github.com/google/guava/tree/master/guava/src/com/google